.vscode/settings.json

```diff
@@ -4,7 +4,7 @@
   "editor.codeActionsOnSave": {
     "source.fixAll": "explicit"
   },
-  "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.tsdk": "packages/commons/node_modules/typescript/lib",
```

Comment:
I wasn't aware about settings.json committed in the repo, is it correct? I got it has not been introduced with this PR.

Reply:
Don't know if it's correct but they are committed at the moment, also on the main/develop branches, and this was pointing to a non existing tsdk 🥲